### PR TITLE
com.github.muriloventuroso.givemelyrics

### PR DESCRIPTION
**Pull request by muriloventuroso**

### CODE REVIEW

File: com.github.muriloventuroso.givemelyrics.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.muriloventuroso.givemelyrics",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri",
```
Comment (bilelmoussaoui) [inline com.github.muriloventuroso.givemelyrics.json]: You do need network access here too?

File: com.github.muriloventuroso.givemelyrics.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "com.github.muriloventuroso.givemelyrics",
+    "runtime": "org.gnome.Platform",
```
Comment (bilelmoussaoui) [inline com.github.muriloventuroso.givemelyrics.json]: You could use freedesktop runtime here.

**Comment by muriloventuroso** [comment]
I requested access to dbus, but the application does not receive the messages. Did I do something wrong?

File: com.github.muriloventuroso.givemelyrics.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "base-version" : "juno",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.muriloventuroso.givemelyrics",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=system-bus",
```
Comment (nedrichards) [inline com.github.muriloventuroso.givemelyrics.json]: This permission is way too broad you need to run your app with --log-system-bus or --log-session-bus as arguments and then whitelist the specific calls you need.

**Comment by nedrichards** [comment]
bot, build com.github.muriloventuroso.givemelyrics
testbot, build com.github.muriloventuroso.givemelyrics

**Comment by nedrichards** [comment]
bot, build com.github.muriloventuroso.givemelyrics
testbot, build com.github.muriloventuroso.givemelyrics

File: com.github.muriloventuroso.givemelyrics.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "com.github.muriloventuroso.givemelyrics",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.muriloventuroso.givemelyrics",
+    "build-options" : {
+        "cxxflags": "--log-session-bus"
```
Comment (nedrichards) [inline com.github.muriloventuroso.givemelyrics.json]: this isn't how you do it. I meant you specify it on the `flatpak run` command line.

**Comment by nedrichards** [comment]
bot, build com.github.muriloventuroso.givemelyrics
testbot, build com.github.muriloventuroso.givemelyrics

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.muriloventuroso.givemelyrics

**Pull request closed without merging**